applications/crossbar/doc/tasks.md

```diff
@@ -0,0 +1,473 @@
+### Backgroud Jobs
+
+Kazoo Tasks enables listing, adding, starting & removing generic background tasks.
+
```

Comment:
You may need to manually include the `tasks.records.json` schema. Use [`cb_api_endpoints:schema_to_table(<<"tasks.records">>).`](https://github.com/2600hz/kazoo/blob/master/applications/crossbar/src/cb_api_endpoints.erl#L10) - that should generate the appropriate output in the shell to paste into this doc.